boards/arm/nucleo_l496zg/nucleo_l496zg_defconfig

```diff
@@ -0,0 +1,41 @@
+CONFIG_ARM=y
+CONFIG_SOC_SERIES_STM32L4X=y
+CONFIG_SOC_STM32L496XG=y
+CONFIG_CORTEX_M_SYSTICK=y
+# 80MHz system clock
+CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=80000000
+
+# enable uart driver
+CONFIG_SERIAL=n
+
+# enable pinmux
+CONFIG_PINMUX=y
+
+# enable GPIO
+CONFIG_GPIO=y
+
+# clock configuration
+CONFIG_CLOCK_CONTROL=y
+# SYSCLK selection
+CONFIG_CLOCK_STM32_SYSCLK_SRC_PLL=y
+# PLL configuration
+CONFIG_CLOCK_STM32_PLL_SRC_HSI=y
+# produce 80MHz clock at PLL output
+CONFIG_CLOCK_STM32_PLL_M_DIVISOR=1
+CONFIG_CLOCK_STM32_PLL_N_MULTIPLIER=20
+CONFIG_CLOCK_STM32_PLL_P_DIVISOR=7
+CONFIG_CLOCK_STM32_PLL_Q_DIVISOR=2
+CONFIG_CLOCK_STM32_PLL_R_DIVISOR=4
+CONFIG_CLOCK_STM32_AHB_PRESCALER=1
+CONFIG_CLOCK_STM32_APB1_PRESCALER=1
+CONFIG_CLOCK_STM32_APB2_PRESCALER=1
+
+# console
+CONFIG_CONSOLE=n
+CONFIG_UART_CONSOLE=n
+
+#enable pwm
+CONFIG_PWM=n
```

Comment:
Comment contradicts the defconfig. Remove? Or add support (see for example nucleo_l476rg).

Reply:
Indeed, please remove.
Having this line here could say that this is available but not enabled, which is not the case as no pin is configured

Reply:
My next move is to support pwm. So, with the next commit it will be resolved. :)

Reply:
Ok, so ideally, group your changes into coherent commits.